containersQa/testlib.bash

```diff
@@ -930,40 +931,42 @@ EOF
 
 }
 
-function setAlgorithmTestsVars {
-  # Turning the tests off for now as there are random failure that are slowing testing for respins.
-  echo $SKIPPED3
-  exit
+function setupAlgorithmTesting {
+  # Skipping the FIPS tests if $OTOOL_CONTAINERQA_RUNCRYPTO is set to anything other than "true"
+  if [ "$OTOOL_CONTAINERQA_RUNCRYPTO" != "true"  ] ; then
+    echo "$SKIPPED8"
+    exit
+  fi
+
   checkAlgorithmsCode=`cat $LIBCQA_SCRIPT_DIR/CheckAlgorithms.java | sed -e "s/'//g"` # the ' characters are escaping and making problems, deleting them here
   cipherListCode=`cat $LIBCQA_SCRIPT_DIR/CipherList.java`
 }
 
+function isHostAndContainerCryptoIdentical() {
+  hostCryptoPolicy=`update-crypto-policies --show`
+  containerCryptoPolicy=`runOnBaseDirBash "update-crypto-policies --show"`
+
+  echo "The crypto policy on host system: $hostCryptoPolicy"
+  echo "The crypto policy in container: $containerCryptoPolicy"
+}
+
+function doesManuallySettingFipsCrash() {
```

Comment:
I suggest changing the method to validateManualSettingFIPSWithNoCrash()

Reply:
I changed the method name and added some more complex logging.